app/page.tsx

```diff
@@ -17,14 +20,26 @@ function Home() {
         </StyledTitle>
       </StyledTitleFrame>
       <Link href="/test">
-        <Image
-          width={170}
-          height={60}
-          src={StartButton.src}
-          alt="스타트 버튼"
-        />
+        <StyledStartButton
+          initial={StartButtonEffect.initial}
+          animate={StartButtonEffect.animate}
+          transition={StartButtonEffect.transition}
+        >
+          <Image
+            width={170}
+            height={60}
+            src={StartButton.src}
+            alt="스타트 버튼"
+          />
+        </StyledStartButton>
       </Link>
-      <StyledSnowManAndTree src={SnowAndTree} alt="눈사람과 트리" />
+      <StyledImageWrapper
+        initial={SnowTreeEffect.initial}
+        animate={SnowTreeEffect.animate}
+        transition={SnowTreeEffect.transition}
```

Comment:
```suggestion
      <StyledImageWrapper
        initial="initial"
        animate="animate"
        transition="transition"
        variants={SnowTreeEffect}
```
로 해도 적용되는 걸로 알고 있는데 되나요 ??

Reply:
원래 이런 식으로 했긴 했는데, transition 부분에서 오류가 계속 뜨더라고요
그럼 transition을 initial와 animate 직접 넣어주는 방식으로 하면 중복되는 코드가 있어서 제가 올린 방식으로 했던거긴한데

지금 보면 차라리 올려주신 방법이 더 좋은거 같기도 합니다🤧
일단 다시 해서 올려보겠습니다

Reply:
상수 구성할 때
```tsx
  initial: { x: -200, transition: { duration: 0.9, ease: defaultEasing } },
  animate: { x: 200, transition: { duration: 0.9, ease: defaultEasing } },
};
```

이런식으로 각각 주어도 좋을 것 같아용